docs/src/assets/scss/docs.scss

```
@@ -1,5 +1,9 @@
 /* docs layout styles */
 
+html, body {
```

I think we only need html here 

```suggestion
html {
```

---

done.